sys/arm64/include/armreg.h

```diff
@@ -62,7 +62,7 @@
 	_val;								\
 })
 #define	WRITE_SPECIALREG(reg, _val)					\
-	__asm __volatile("msr	" __STRING(reg) ", %0" : : "r" ((uint64_t)_val))
```

Comment:
Hm, did this get accidentally changed in morello-dev relative to dev or in dev relative to upstream? I know the whitespace has always been dodgy in these macros but I forget how exactly.

Reply:
So I didn't include this bit when upstreaming because the capability one was inconsistent.  I also looked in various arm64 include headers and there is a mix of putting the space and omitting the space.  I think they should all use the space personally, but this is how it is upstream (no space).

Reply:
Yeah there doesn't seem to be consistency upstream (riscv also varies depending on where you look). I'm personally in the no space camp, but it should at least be consistent.